### ISSUE TITLE
Roadmap for logout implementation in SAML connector?

### ISSUE THREAD

**Issue by niamhcahill** [dexidp/dex#1600]
Are there any roadmap plans to support Single Log Out methods in the SAML connector? We can set the session timeout values to be low on the SAML identity provider, but an actual logout option would be great.

**Comment by sandeepbangera** [comment]
Yes it will be great to be able to explicitly log out SAML OIDC sessions from Dex.
